src/person/controllers/developer_controller.py

```diff
@@ -0,0 +1,51 @@
+from random import randint
+
+from api.utils import failure_response
+from api.utils import success_response
+from django.contrib.auth.models import User
+from person.models import Person
+from rest_framework import status
+from rest_framework.authtoken.models import Token
+
+
+class DeveloperController:
+    def __init__(self, request, data, serializer):
+        self._request = request
+        self._data = data
+        self._serializer = serializer
+
+    def create_person(self, person_data):
+        """Creates new Person object from `person_data.`"""
+        person = Person(**person_data)
+        person.save()
+        return person
+
+    def create_user(self, user_data):
+        """Creates new user (Django auth) from `user_data.`"""
+        return User.objects._create_user(**user_data)
+
+    def process(self):
+        """Creates new user and person if user_id not provided in body. Otherwise, return access token for valid user."""
+        status_code = status.HTTP_200_OK
+        user_id = self._request.GET.get("id")
```

Comment:
shouldn't be getting through `GET` params tho... the get param should be piped in from the URL
`asdfasd/<an_id_name_you_define>/`

get(request,an_id_name_you_define)

but i guess this is... fine? 

since it's a controller.. but just note that reading it in directly in the get/post/delete methods is probs more traditional

Reply:
cast it to an int here

Reply:
i said that but then i deleted my comment saying that. it's bc she's going to be checking if it's none, and casting a none to int is not expected